endpoint-edgecases/net_version.md

```diff
@@ -0,0 +1,9 @@
+# `net_version`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |net_version MUST return the current network id of the client as type string|
```

Comment:
What does this default to?

Reply:
I believe there is no type string -> we need to specify either QUANTITY or DATA here? or it string actually?